kolibri/plugins/coach/assets/src/actions.js

```diff
@@ -224,10 +224,22 @@ function showContentUnavailable(store) {
   store.dispatch('CORE_SET_TITLE', 'Content Unavailable');
 }
 
+
+// - - - - - Action for Coach Exercise Render Page - - - - - -
+
+function showCoachExerciseRenderPage(store) {
+  store.dispatch('CORE_SET_PAGE_LOADING', true);
+  store.dispatch('SET_PAGE_NAME', Constants.PageNames.COACH_EXERCISE_RENDER_PAGE);
+  store.dispatch('CORE_SET_PAGE_LOADING', false);
+  store.dispatch('CORE_SET_TITLE', 'Coach Exercise Render');
```

Comment:
I think the title of this page should be something like 'Exercise Detail View'. Also, we need any interface text, currently, to be in Vue components for it to be translatable.